bazel/repository_locations.bzl

```diff
@@ -40,24 +40,37 @@ USE_CATEGORIES = [
 # Components with these use categories are not required to specify the 'cpe' annotation.
 USE_CATEGORIES_WITH_CPE_OPTIONAL = ["build", "test", "other"]
 
+BAZEL_COMPDB_VERSION = "0.4.5"
+BAZEL_GAZELLE_VERSION = "0.19.1"
+BAZEL_TOOLCHAINS_VERSION = "3.4.0"
```

Comment:
Should we follow the format of https://github.com/envoyproxy/envoy/blob/master/api/bazel/repository_locations.bzl#L1-L23 where both the version / Git SHA and SHA256 are editable in pairs? Without doing this to bump versions you need to edit the version at the top of the file and then search for the entry below to update the SHA256

Reply:
Yeah, this seems alright. With this change, are you good with this metadata as a start?

Reply:
I switched to something a bit cleaner (and will update api/bazel/repository_locations.bzl to this in the future) - using string interpolation to allow this to be a regular dictionary field. PTAL.